bin/create.js

```diff
@@ -57,6 +74,10 @@ var tokenOptions = {
         description: 'pre-compile a local file using the indicated library (for now only `babel` is supported and will read `.babelrc` if present)',
         type: 'string',
     },
+    auth: {
+      description: 'set webtask permisions',
```

Comment:
Think this used to be where you could specify emails and domains, you're right it's no longer needed.